src/config.ts

```diff
@@ -45,6 +45,11 @@ export interface IntegrationConfig extends IntegrationInstanceConfig {
    * The provider API client secret used to authenticate requests.
    */
   apiSecret: string;
+
+  /**
+   * For use in tests only. Avoid ingesting entire account's worth of applications to reduce PollyJS Recording size
+   */
+  targetApplication?: string;
```

Comment:
the PollyJS recording when using our test account of approx 15K findings was about **35MB**! 😱 
this is only for typings, and it is not possible to set this value as there is no environment-mapping option for it locally, nor will there be a configuration option for it in the managed integration.